### PR TITLE
@kanaabe => Artist follows

### PR DESCRIPTION
**Pull request by eessex**
Not super excited about using jquery, but it seemed the only way to get this working with our text components. 

- Sets up user's followed artists in App and InfiniteScrollArticle
- Sets up FollowButtons when component mounts and sync's with users follows

### CODE REVIEW

**Comment by kanaabe** [comment]
Hmm I was expecting this setup/logic to happen in Reaction since the Text component exists there. Can you take me through your reasoning for putting it in Force? I believe that this works, but I guess it doesn't make sense to me that a component can't fully work on its own. I think it might be worth taking a stab at doing this in Reaction if you're up for it.

This could be a good example to follow: https://github.com/artsy/reaction/blob/master/src/Components/Follow.tsx

When you have that gut feeling that using jQuery is painful, it's because there's always another way!

**Comment by eessex** [comment]
As far as having to use jQuery -- we are using dangerouslySetInnerHTML, so even if we did it in reaction, it would probably require jQuery to parse the html and insert.

**Comment by eessex** [comment]
Totally down to try and do this in Reaction, but I think I'll need to pair with someone to figure out Relay, which I haven't used before -- the follow component is broken in storybooks and I wasn't able to get it running.  Cab built it, but maybe someone on collector experience knows better how to get it working.

**Comment by kanaabe** [comment]
I wouldn’t want to introduce jQuery to Reaction either because there’s always the plain DOM API. I’m unfortunately not well-versed in Relay either, but this feels like a great opportunity to dive into it since this exact functionality is already built in Reaction. I’m happy to pair on it together!

However, I do understand that we have a deadline and I don’t want things like this blocking us. Maybe try it out for 1 hour on a Friday afternoon and see where you go. For now I’ll merge this with the understanding that this is incurring tech debt and that we’ll refactor it the first opportunity we have.

**Comment by damassi** [comment]
I would avoid jQuery as there are many light-weight alternatives and browsers are now up to speed. Its really big size-wise, too! If you need a specific jQuery function theres typically a stand-alone npm module you can find and use.

File: desktop/apps/article2/components/FollowButton.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  if (CurrentUser.orNull()) {
+    return new Following(null, {kind: 'artist'})
+  }
+}
+
+export const setupFollowButtons = (following) => {
+  const artists = map($('.artist-follow'), artist => {
+    return artist.getAttribute('data-id')
+  })
+  map(artists, artist => {
```
Comment (damassi) [inline desktop/apps/article2/components/FollowButton.js]: Don't need map / reduce / filter, etc -- all native methods and we're using babel-polyfill, so cross-browser isn't a concern: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array

**Pull request merged**